metaserver/src/main/scala/scala/meta/languageserver/compiler/CompletionProvider.scala

```diff
@@ -32,11 +33,28 @@ object CompletionProvider extends LazyLogging {
         isUsedLabel += label
         items += CompletionItem(
           label = label,
-          detail = Some(r.sym.signatureString)
+          detail = Some(r.sym.signatureString),
+          kind = completionItemKind(compiler)(r)
         )
       }
     }
     CompletionList(isIncomplete = false, items = items.result())
   }
 
+  private def completionItemKind(
+      c: Global
+  )(r: c.CompletionResult#M): Option[Int] = {
+    if (r.sym.isPackage) Some(CompletionItemKind.Module)
```

Comment:
Would it make sense to use something else like `Reference` just to distinguish it from objects? Seems like `Namespace` is missing :/

Reply:
`Reference` looks very strange on vscode, but I was thinking to simply fallback to `Value` for everything that doesn't have a proper fit in `CompletionItemKind`

Reply:
I think I'll just

Reply:
I also wouldn't worry too much for now, I'm sure the LSP will evolve in this sense and v4 may include more symbols.